docs/workflow/using-dotnet-cli.md

```diff
@@ -11,12 +11,16 @@ For other walkthroughs see:
 
 ## Prerequisites
 
-1. Successfully built this repository and thus have files of the form shown below. From now on we call this folder NuGet package folder.
+All paths in examples below are Windows-style but the procedure is otherwise exactly the same on Unix.
+
+1. Successfully built this repository including the shared framework package and thus have files of the form shown below. From now on we call this folder your NuGet package folder.
 
 ```
-    artifacts\packages\<configuration>\Shipping\
+    <your-repo-root>\artifacts\packages\<configuration>\Shipping\
 ```
 
+If you don't have this folder, you may have built binaries but not packages. Try building from the root with a command like `build.cmd clr+libs+host+packs -rc release`.
```

Comment:
```suggestion
If you don't have this folder, you may have built binaries but not packages. Try building from the root with a command like `build.cmd clr+libs+host+packs -c release`.
```
The use the runtime in your app, you typically want to build everything as release.